cms_product_form/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+CMS Product Form
+================
+
+Basic content type product form.
```

Comment:
As you provide 2 forms:
1. `s/form/forms`
2. list the forms and their controllers